### PR TITLE
added loggings

### PR DESCRIPTION
**Pull request by vglalitha**

### CODE REVIEW

File: dropwizard-demo/pom.xml [hunk 1]
```diff
@@ -51,7 +51,27 @@
...
+            <version>3.12.4</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <version>1.7.32</version>
+        </dependency>
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
```
Comment (PoojaMohite-Khoros) [inline dropwizard-demo/pom.xml:66]: Y this?

File: dropwizard-demo/test/com/test/ControllerTest.java [hunk 2]
```diff
@@ -32,30 +32,6 @@ public void setUp() {
         tweetPost = Mockito.mock(Controller.class);
     }
 
-    @Test
-    public void testcase1_getTweets() {
```
Comment (PoojaMohite-Khoros) [inline dropwizard-demo/test/com/test/ControllerTest.java:36]: Y r u deleting test

File: dropwizard-demo/test/com/test/ControllerTest.java [hunk 3]
```diff
@@ -103,28 +79,6 @@ public void testcase_sendTweet_success() {
         Assert.assertTrue(T);
     }
 
-    @Test
-    public void testcase_searchTweets() throws TwitterException {
-        when(brsConfiguration.configurationBuilder()).thenReturn(new ConfigurationBuilder());
-        ArrayList<String> tweet = new ArrayList<String>();
-        tweet.add("hlo");
-        tweet.add("hi");
```
Comment (PoojaMohite-Khoros) [inline dropwizard-demo/test/com/test/ControllerTest.java:111]: here too , whats happening

File: dropwizard-demo/src/main/java/com/dep/resource/Controller.java [hunk 4]
```diff
@@ -62,19 +43,24 @@ public Response GetTweets() {
...
     public String healthCheck() {
+        log.info("got into healthCheck");
+        log.trace("home method access");
         return "Ping Received at " + new Date();
     }
 
     @POST
     @Path("/tweetAgain")
     public Response tweetAgain(Request request) throws TwitterException {
+        //logger.info("got into post");
```
Comment (PoojaMohite-Khoros) [inline dropwizard-demo/src/main/java/com/dep/resource/Controller.java:54]: wer r u using the log?

**Review by PoojaMohite-Khoros — CHANGES_REQUESTED** [review]
Work incomplete, can do better, 
Please address reviews

File: dropwizard-demo/test/com/test/ControllerTest.java (hunk 2 above)
Comment (vglalitha) [reply dropwizard-demo/test/com/test/ControllerTest.java:36]: This testcases are related to duplicate code of my GET method. This  testcases is getting repeated , so I have deleted this testcases.

File: dropwizard-demo/test/com/test/ControllerTest.java (hunk 3 above)
Comment (vglalitha) [reply dropwizard-demo/test/com/test/ControllerTest.java:111]: This testcases are related to duplicate code of my GET method. This testcases is getting repeated , so I have deleted this testcases.

File: dropwizard-demo/pom.xml (hunk 1 above)
Comment (vglalitha) [reply dropwizard-demo/pom.xml]: This is unnecessary dependency , i have deleted this dependency .

File: dropwizard-demo/src/main/java/com/dep/resource/Controller.java (hunk 4 above)
Comment (vglalitha) [reply dropwizard-demo/src/main/java/com/dep/resource/Controller.java]: addressed the reviews

**Review by PoojaMohite-Khoros — APPROVED** [review]
💯

**Pull request merged**